.travis.yml

```diff
@@ -1,5 +1,20 @@
 language: java
 
-os:
-  - linux
-  - osx
+dist: trusty
+
+matrix:
+  include:
+  - os: linux
+    jdk: oraclejdk8
+  - os: linux
+    jdk: oraclejdk9
+  - os: linux
+    jdk: openjdk10
+  - os: linux
+    jdk: openjdk11
+  - os: linux
+    jdk: openjdk12
+  - os: osx
+    jdk: oraclejdk8
+  - os: osx
+    jdk: openjdk11
```

Comment:
is there `openjdk12` available on `osx`?

Reply:
Not according to this page, but I didn't test: https://docs.travis-ci.com/user/reference/osx/#macos-version

My intention was to minimize the number of macOS jobs. Otherwise, we could do a full matrix, assuming all JDKs are supported.